### ISSUE TITLE
LearningActivityChip background contrast fails a11y check

### ISSUE THREAD

**Issue by nucleogenesis** [learningequality/kolibri#8919]
## Observed behavior

The light gray used as the background on LearningActivityChip is too light and can be barely visible in some cases. I suspected this while testing on Chrome, but for some reason it became glaringly obvious when I couldn't see the background at all while testing on BrowserStack IE11. I can barely see it on one of my screens and almost cannot see it at all on the other.

![image](https://user-images.githubusercontent.com/6356129/145936574-5da76ae2-cd0b-4dda-beb1-20b8ba2e7c43.png)

![image](https://user-images.githubusercontent.com/6356129/145936596-1dde0f22-4807-4d01-9b02-4a9af0427a03.png)

The background color is hard-coded (as noted in #8899) and it ought to instead use some of our [theme colors](https://kolibri-design-system.netlify.app/colors/).

However, as the icons are also colored, whatever background color we choose for these chips (if any) should also pass a11y check against the newly decided color. For example, we will want `tokens.video` to pass as a foreground against whatever darker grey is decided upon.
 

## Expected behavior

Colors used in Kolibri meet a11y standards

## User-facing consequences

Color intended to provide visual spacing to elements will not be visible. This isn't really a huge issue but it may ultimately not reflect the desired design.

**Comment by MisRob** [comment]
Just a note that the background color won't be hardcoded anymore after we merge https://github.com/learningequality/kolibri/pull/9011, however, we still need to resolve the insufficient contrast issue.

**Comment by nucleogenesis** [comment]
Per Radina - background on background is less an a11y concern as long as the text contrast is accessible. Less an a11y issue but perhaps a design concern?

Tomiwa notes that we could play with border or darker gray -- but also that it works alright as-is.
